src/main/java/net/sf/mzmine/modules/visualization/peaklisttable/PeakListTablePopupMenu.java

```diff
@@ -143,7 +143,7 @@ public PeakListTablePopupMenu(final PeakListTableWindow window, PeakListTable li
     showSpectrumItem = GUIUtils.addMenuItem(showMenu, "Mass spectrum", this);
     show2DItem = GUIUtils.addMenuItem(showMenu, "Peak in 2D", this);
     show3DItem = GUIUtils.addMenuItem(showMenu, "Peak in 3D", this);
-    showMSMSItem = GUIUtils.addMenuItem(showMenu, "Most intense MS/MS", this);
+    showMSMSItem = GUIUtils.addMenuItem(showMenu, "MS/MS (single/multiple)", this);
```

Comment:
Great addition Robin! 
I am not 100% sure if I like the new label of this button. I think the information that the method selects the most intense MS/MS scan is more useful compared to the applicability on multiple rows. Maybe changing back to "Most intense" and adding "single/multiple row selection" to the tool tip? What do you think?

Reply:
I agree